arbnode/dataposter/data_poster.go

```diff
@@ -770,7 +772,7 @@ var DefaultDataPosterConfig = DataPosterConfig{
 	UseNoOpStorage:         false,
 	LegacyStorageEncoding:  true,
 	Dangerous:              DangerousConfig{ClearDBStorage: false},
-	ExternalSigner:         ExternalSignerCfg{Enabled: false, Method: "eth_signTransaction"},
+	ExternalSigner:         ExternalSignerCfg{},
```

Comment:
I think we should still keep the default method as eth_signTransaction as that seems to be the most common method name (also applies to `DefaultDataPosterConfigForValidator`)

Reply:
Done. ( I guess you meant `TestDataPosterConfig` not  `DefaultDataPosterConfigForValidator`)

Reply:
Yeah, I got confused by the github diff

![image](https://github.com/OffchainLabs/nitro/assets/6006498/a3042707-d3be-4cd9-8160-0080c11378ae)